app/views/events/edit.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Edit Event</h1>
+
+<%= render 'form' %>
+
+<%= link_to 'Back', events_path %>
```

Comment:
Two things:

1.) This commit message doesn't give any hints that we are creating a view. I don't think it should be split into another commit because we would have tests broken on this commit but maybe we could mention it in our PR message.

2.) Linking to back. We don't know how the user got here we shouldn't assume they came from `/events/:id`